File: python/b11005.py

# 11005: 진법 변환 2

inputs = input().split(" ")
N = int(inputs[0])
B = int(inputs[1])

result = ""

while N > 0:
	data = N % B
	if data > 9:
		data = chr(data + ord('A') - 10)
	result += str(data)
	N = int(N / B)

print(result[::-1])

File: c/b11005.c
#include <stdio.h>
#include <string.h>

// 11005: 진법 변환

int main(void) {
	int B, data, i, j;
	long long N;
	char result[36];

	scanf("%lld %d", &N, &B);

	i = 0;
	while (N > 0) {
		data = N % B;
		if (data > 9) {
			data += 'A' - 10;
		} else {
			data += '0';
		}
		result[i++] = data;
		N /= B;
	}

	for (j = i-1; j >= 0; j--) {
		printf("%c", result[j]);
	}
}